### PR TITLE
Room insights initial commit

### PR DESCRIPTION
**Pull request by Jharv94**
# Add app insights into room., 204

## change proposal
(~) Modified, (+) Added, (-) Removed, (R) Replaced
----------------------------------------------------------------------
~ "formatting" to docker-compose file to "organize" what containers are "grouped" together

[+] APP_INSIGHTS environment variable inside appsettings.json
[+] ApplicationInsights.Aspnetcore to room.csproj file
[+] Application Insights Telemetry has been added to Configureservices in Startup
[+] Live metrics for Room Api to App Insights
[+] Health Checks for Room Api

## reviewer

- @jamesgoldsmith8 
- @fredbelotte 

## contributing

this project is maintained under the terms of the [Contribution Guidelines][contributing].

[contributing]: https://github.com/fredbelotte/alpine/blob/master/.github/CONTRIBUTING.md 'contribution guidelines'

### CODE REVIEW

File: manifest.docker.yaml [hunk 1]
```diff
@@ -115,7 +115,7 @@ services:
     image: postgres:alpine
     networks:
       - locally
-
+######################################################################################
```
Comment (jamesgoldsmith8) [inline manifest.docker.yaml]: remove

File: manifest.docker.yaml [hunk 2]
```diff
@@ -87,7 +87,7 @@ services:
     image: postgres:alpine
     networks:
       - locally
-
+#####################################################################################
```
Comment (jamesgoldsmith8) [inline manifest.docker.yaml]: remove

File: manifest.docker.yaml [hunk 3]
```diff
@@ -59,7 +59,7 @@ services:
     image: postgres:alpine
     networks:
       - locally
-
+###########################################################################
```
Comment (jamesgoldsmith8) [inline manifest.docker.yaml]: remove

File: manifest.docker.yaml [hunk 4]
```diff
@@ -31,7 +31,7 @@ services:
     image: postgres:alpine
     networks:
       - locally
-
+###############################################################################
```
Comment (jamesgoldsmith8) [inline manifest.docker.yaml]: remove

File: room/src/Revature.Room.Api/Revature.Room.Api.csproj [hunk 5]
```diff
@@ -25,4 +25,9 @@
     <ProjectReference Include="..\Revature.Room.DataAccess\Revature.Room.DataAccess.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.12.0" />
+    <PackageReference Include="Microsoft.ApplicationInsights.PerfCounterCollector" Version="2.12.0" />
```
Comment (jamesgoldsmith8) [inline room/src/Revature.Room.Api/Revature.Room.Api.csproj]: why?

Comment (Jharv94) [reply room/src/Revature.Room.Api/Revature.Room.Api.csproj]: Per documentation this allows us to get Live metrics stream.
Seen here -> [https://docs.microsoft.com/en-us/azure/azure-monitor/app/live-stream#get-started](url)

Comment (jamesgoldsmith8) [reply room/src/Revature.Room.Api/Revature.Room.Api.csproj]: sounds good

**Pull request closed without merging**